openfisca_aotearoa/variables/rates_rebates.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+
+# Import from openfisca-core the common python objects used to code the legislation in OpenFisca
+from openfisca_core.model_api import *
+# Import the entities specifically defined for this tax and benefit system
+from openfisca_aotearoa.entities import Propertee, Person
+from numpy import clip
+
+
+# This is a 'temporary' variable used specifically for the rates rebates
+# Due to the structure not yet being clear, this may need to be moved, renamed.
+class dependants(Variable):
+    value_type = int
+    entity = Person
+    definition_period = YEAR
+    label = u"Number of Persons classified as dependant on the entity Person for the purposes of rates rebates"
```

Comment:
> Adding the “on the entity Person” is not very useful here as it can be read easily in the code, and the information is [displayed in the Legislation explorer](http://213.32.89.165/dependants) as well.

Reply:
> Agreed